README.md

```diff
@@ -1,13 +1,19 @@
 # AngularElementApp
 
+## Technical stack
+
 Angular element app example.
 Build an angular application into a generic web component that can be uses in any html page.
 
+## Purpose of this library
+
+Add a menu widget to redirect to the centralized connection of other applications, editable in the configuration file: image, text, links
```

Comment:
Added in you frontend, this widget will allow you users to nagivate to other stelliance's companies web applications using SSO.

Reply:
ok

Reply:
ok